app/models/secret_storage.rb

```diff
@@ -50,6 +50,43 @@ def self.keys
     end
   end
 
+  class HashicorpVault
+
+    SECRET_BACKEND ='secret/'.freeze
```

Comment:
the path there actually tells vault what storage backend it's using.  the prefix secret/ is puts the stuff into that secret engine in vault.  I'll rename it to make it more clear.